src/streamlink/plugins/bfmtv.py

```diff
@@ -1,4 +1,5 @@
 """
+$description BFM: National & regional news, weather and business. 01TV: technology news and topics. Both based in France.
```

Comment:
> BFM:
> 01TV:

don't split it, you could use less details

Reply:
https://en.wikipedia.org/wiki/NextRadioTV